### PR TITLE
Fix broken table migration workflow links in README

### PR DESCRIPTION
**Pull request by qziyuan**
1. Fix broken #table-migration-workflow links in README. 
2. Add mermaid flow chart to demonstrate the table migration workflows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ After [installation](#install-ucx), ensure to [trigger](#ensure-assessment-run-c
 so that you'll be able to [scope the migration](docs/assessment.md) and execute the [group migration workflow](#group-migration-workflow).
 
 The [README notebook](#readme-notebook), which can be found in the installation folder contains further instructions and explanations of the different ucx workflows & dashboards.
-Once the migration is scoped, you can start executing the [table migration workflow](#table-migration-workflow).
+Once the migration is scoped, you can start executing the [table migration workflow](#Table-Migration).
```
Comment (JCZuurmond) [inline README.md:11]: Let's change the "table migration workflow" to "table migration process" as it is more than just one workflow like the assessment:

```suggestion
Once the migration is scoped, you can start with the [table migration process](#Table-Migration).
```

Can you update the others as well?

File: README.md [hunk 2]
```diff
@@ -536,7 +536,48 @@ We also add a `upgraded_from_workspace_id` property to the upgraded object, to i
 
 The upgrade process can be triggered using the `migrate-tables` [UCX command](#migrate-tables-command)
 
-Or by running the "Migrate Tables" workflow deployed to the workspace.
+Or by running the table migration workflows deployed to the workspace.
+
+##### table migration workflows
```
Comment (JCZuurmond) [inline README.md:541]: Move this to the top or end of the `Table Migration` process to show an overview to users.

```suggestion
#### Table migration workflows
```

**Review by JCZuurmond — COMMENTED** [review]
Thanks for the changes, have two comments

File: README.md (hunk 1 above)
Comment (qziyuan) [reply README.md]: Done

**Review by JCZuurmond — APPROVED** [review]
thanks!

**Review by FastLee — APPROVED** [review]

**Pull request merged**